source/_integrations/fireservicerota.markdown

```diff
@@ -0,0 +1,286 @@
+| `responded_at` | Time responded.                                             |
+| `start_time` | Incident repsonse start time.                                             |
+| `channel` | Channel.                                             |
+| `status` | Status of repsonse, e.g. 'pending'.                                             |
+| `reported_status` | Reported status, e.g. 'shown_up'.                                             |
+| `arrived_at_station` | 'true' or 'false'.                                             |
+| `available_at_incident_creation` | 'true' or 'false'.                                             |
+| `active_duty_function_ids` | Active function id's, e.g. '540'.                                             |
+
+
+## Advanced Configuration
+
+Practical use can be sounding an alarm and/or blinking lights when an emergency incident is received, use text to speech to play incident details while getting dressed, respond with an acknowledge using a door-sensor leaving house or a button or something else connected to Home Assistant to let your teammates know you are underway.
+
+### FireServiceRota Dashboard
+
+If you want to have your browser to function as audio player, and have screen control you need to install the 'browser_mod' integration (via HACS)
+
+Set the device alias below to match your media player id.
+
+You also need to install the following plugins for Lovelace (again HACS is your friend)
+- 'custom:layout-card'
+- 'custom:bignumber-card'
+- 'custom:vertical-stack-in-card'
+- 'custom:home-feed-card'
+
+{% raw %}
+
+```yaml
+browser_mod:
+  devices:
+    3184e47f_bac9b576:  # <--- change this id to your browser id
+      name: bwr-nesthub
+
+sensor:
+  - platform: template
+    sensors:
+      incidents_speech:
+        friendly_name: "Speech Sensor"
+        value_template: "{{ state_attr('sensor.incidents', 'message_to_speech_url') }}"
+
+input_boolean:
+  bwr_tts:
+    name: Text to Speech
+    icon: mdi:text-to-speech
+
+homeassistant:
+  customize:
+    light.bwr_nesthub:
+      friendly_name: 'Display On/Off'
+
+automation:
+  - alias: Play Incident Speech
+    trigger:
+      platform: state
+      entity_id: sensor.incidents
+    action:
+      - service: media_player.play_media
+        data_template:
+          entity_id: media_player.bwr_nesthub
+          media_content_id: >
+              {{ states('sensor.incidents_speech') }}
+          media_content_type: 'audio/mp4'
+      - service: homeassistant.turn_on
+        data_template: 
+          entity_id: >
+              light.bwr_nesthub
+      - service: timer.start
+        data:
+          entity_id: timer.bwr_nesthub
+    condition:
+      condition: and
+      conditions:
+      - condition: time
+        after: '06:30:00'
+        before: '23:00:00'
+      - condition: template
+        value_template: "{{ not is_state('sensor.incidents_speech', 'None') }}"
```

Comment:
Why are we checking for this?

Reply:
The attribute can be missing or does not yet contain an URL, to prevent mediaplayer filling log when this happens.
But I simplified/rewrote the automation.